doc/nrf/known_issues.rst

```diff
@@ -483,25 +484,25 @@ KRKNWK-6408: ``diag`` command not supported
 Zigbee
 ======
 
-.. rst-class:: v1-9-1 v1-9-0
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0
 
 KRKNWK-12937: Activation of Sleepy End Device must be done at the very first commissioning procedure for light switch sample
    After programming the :ref:`Zigbee light switch <zigbee_light_switch_sample>` sample and its first commissioning, Zigbee End Device joins the Zigbee network as a normal End Device. Pressing **Button 3** does not switch the device to the Sleepy End Device configuration.
 
    **Workaround:** Keep **Button 3** pressed during the first commissioning procedure.
 
-.. rst-class:: v1-9-1 v1-9-0
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0
```

Comment:
Fixed in 2.0.0, as per https://projecttools.nordicsemi.no/confluence/display/KRKNWK/Release+notes+and+known+issues+2.0.0 from @maciejpietras